iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -46,10 +44,13 @@ class ConversationListViewController: FullScreenViewController,
     // Custom Input Accessory View
     lazy var messageInputAccessoryView: ConversationInputAccessoryView = {
         let inputView: ConversationInputAccessoryView = ConversationInputAccessoryView.fromNib()
-        inputView.delegate = self
+        inputView.delegate = self.swipeInputDelegate
         inputView.textView.restorationIdentifier = "list"
         return inputView
     }()
+    lazy var swipeInputDelegate = SwipeableInputAccessoryMessageSender(viewController: self,
+                                                                       collectionView: self.collectionView,
+                                                                       isConversationList: true)
```

Comment:
Here's the delegate that handles showing the drop zone and adjusting the collection view offsets, etc.